src/io/flutter/run/OpenDevToolsAction.java

```diff
@@ -50,21 +51,29 @@ public void update(@NotNull final AnActionEvent e) {
   public void actionPerformed(@NotNull final AnActionEvent event) {
     FlutterInitializer.sendAnalyticsAction(this);
 
-    if (event.getProject() == null) {
+    Project project = event.getProject();
+    if (project == null) {
       return;
     }
 
-    final DevToolsManager devToolsManager = DevToolsManager.getInstance(event.getProject());
+    final DevToolsManager devToolsManager = DevToolsManager.getInstance(project);
 
-    final Optional<FlutterApp> appOptional = FlutterUtils.findFlutterAppFromProject(event.getProject());
+    final Optional<FlutterApp> appOptional =
+      FlutterApp.allFromProjectProcess(project).stream().filter((FlutterApp app) -> app.getProject() == project).findFirst();
```

Comment:
This code still has the problem that we are finding all FlutterApps for the project and picking one rather than usinig the app for the app being debugged. Can we pick the right app?

Reply:
Ah yeah, I can pass the app from where `OpenDevToolsAction` is created.